salt/modules/lxc.py

```diff
@@ -1037,7 +1097,9 @@ def select(key, default=None):
     does_exist = exists(name)
     to_reboot = False
     remove_seed_marker = False
-    if clone_from:
+    if does_exist:
+        pass
```

Comment:
if you read the rest of the code, it is more for the code understanding at a whole.